fcc-test-suite-template-master(1)/fcc-test-suite-template-master/css/style.css

```diff
@@ -0,0 +1,56 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    color: rgb(177, 99, 99);
+}
+body {
+    text-align: center;
+    background-color: rgb(45, 51, 51);
+}
+#main {
+    margin-top: 70px;
+    background-color: rgb(59, 57, 57);
```

Comment:
I think your CSS is great, but please work on the color pallet of your design

This is how your design look. I think it would look better if you used a lighter `background-color `or used colors that match so that it can be appealing to the user.

![image](https://user-images.githubusercontent.com/69511010/171340892-73dcf497-c9f9-4c9c-9293-45c56017d64e.png)